core/config/config.go

```diff
@@ -364,10 +364,11 @@ func migrateAccessTokens(ctx context.Context, db pg.DB, sdb *sinkdb.DB) error {
 		case "network":
 			grant.Policy = "crosscore"
 		}
-		err = sdb.Exec(ctx, store.Save(ctx, &grant))
-		if err != nil {
-			return errors.Wrap(err)
-		}
+		stores = append(stores, store.Save(ctx, &grant))
```

Comment:
Could this produce multiple writes to the same key in this list? If so, that would be an [error](https://github.com/chain/chain/blob/ce5df096c4416342a5f5a31e9a3d2d4ad43cbaec/database/sinkdb/sinkdb.go#L76-L87). Should prob add a test case for that scenario.